tests/nested/manual/uc-update-assets-secure/task.yaml

```diff
@@ -50,9 +55,12 @@ prepare: |
   cp shim.efi.out pc/shim.efi.signed
   tests.nested secboot-sign file pc/shim.efi.signed "${SNAKEOIL_KEY}" "${SNAKEOIL_CERT}"
 
+  tests.nested secboot-remove-signature pc/grubx64.efi
```

Comment:
This is because the old signatures are broken. So to be less confusing. However, I have noticed that `tests.nested secboot-sign file` removes old signatures. So it is not needed. But that implies that we cannot sign with 2 signatures so the code after was wrong. I am going to fix that. I happens to work because the snakeoil is in db, so the grub signed with it still works... It is a bit annoying. But it will do.